stepbystep/index.html

```diff
@@ -0,0 +1,147 @@
+    <div class="columns">
+      <div class="two-thirds column centered">
+        <section class="header text-center">
+          <h1 class="title">WebAssembly</h1>
+          <a class="btn btn-outline" href="../">Overview</a>
+          <a class="btn btn-outline" href="../demo/">Demo</a>
+          <a class="btn btn-outline" href="https://github.com/WebAssembly/design">Design</a>
+          <a class="btn btn-outline" href="https://github.com/WebAssembly/spec">Specification</a>
+          <a class="btn btn-outline" href="https://www.w3.org/community/webassembly/">Community Group</a>
+          <a class="btn btn-outline" href="./">Step by Step</a>
+        </section>
+      </div>
+    </div>
+    <div class="columns">
+      <div class="three-fourths column centered">
+        <h2>Step by Step Setup</h2>
+        <p>
+          This page aims to provide complete, working step-by-step instructions
+          to compile a simple program directly to WebAssembly.
+        <p>
+        <h3>Install the correct version of cmake</h3>
+        <p>
+          To compile the rest of the tools, cmake is required. Specifically,
+          LLVM requires cmake 3.4.3 or higher. If you already have cmake 3.4.3
+          or higher, or are able to install or upgrade cmake through your
+          package manager, you can skip this section.
+        </p>
+        <p>
+          The latest version of cmake comes precompiled and can be downloaded
+          with the following commands:
+        </p>
+        <pre>
+          $ curl https://cmake.org/files/v3.6/cmake-3.6.0-Linux-x86_64.tar.gz \
+                 -o cmake-3.6.0-Linux-x86_64.tar.gz
+          $ gunzip cmake-3.6.0-Linux-x86_64.tar.gz
+          $ tar -xf cmake-3.6.0-Linux-x86_64.tar
+          $ PATH=`pwd`/cmake-3.6.0-Linux-x86_64/bin:$PATH
+        </pre>
+        <h3>Install the correct version of LLVM</h3>
+        <p>
+          To compile to WebAssembly, emscripten requires a "vanilla" build of LLVM
+          with WebAssembly support, which is only available through a build flag.
+          LLVM also needs to be integrated with clang, which is easiest to do at
+          compile time.
+        </p>
+        <p>
+          The <a href="http://clang.llvm.org/get_started.html">clang getting started page</a>
+          provides an excellent step-by-step guide to checking out and compiling LLVM with clang.
+          There is one change we need to make, however: when calling cmake, we need to add
+          the following flag: <code>-DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly</code>
+        </p>
+        <pre>
+          $ svn co http://llvm.org/svn/llvm-project/llvm/trunk llvm
+          $ cd llvm/tools
+          $ svn co http://llvm.org/svn/llvm-project/cfe/trunk clang
+          $ cd clang/tools
+          $ svn co http://llvm.org/svn/llvm-project/clang-tools-extra/trunk extra
+          $ cd ../../../projects
+          $ svn co http://llvm.org/svn/llvm-project/compiler-rt/trunk compiler-rt
```

Comment:
These should probably point at the git mirrors instead, since they're https: https://github.com/llvm-mirror